Automattic/woocommerce-payments
# HACK week: migration to TypeScript (client/card-readers/list)

Migration to TS of `client/card-readers/list` folder component.

#### Testing instructions

- Enable the Card Readers menu item (you can force `calculate_card_present_eligibility` to return true [in the server](https://github.com/Automattic/woocommerce-payments-server/blob/e481ea6d1c570d5a6d3872c4446bb20d1f7b57f5/server/wp-content/rest-api-plugins/endpoints/wcpay/services/class-account-service.php#L1052))
- Update the `_wcpay_feature_card_readers` option to `1`.
- Go to Payments > Card Readers
- You should see the list of available card readers
